mailbox/api/src/main/java/org/apache/james/mailbox/model/Cid.java

```diff
@@ -20,15 +20,21 @@
 package org.apache.james.mailbox.model;
 
 
+import org.apache.commons.lang.StringUtils;
+
 import com.google.common.base.Objects;
 import com.google.common.base.Preconditions;
 
 public class Cid {
 
     public static Cid from(String cidAsString) {
         Preconditions.checkNotNull(cidAsString);
-        Preconditions.checkArgument(!cidAsString.isEmpty(), "'cidAsString' is mandatory");
-        return new Cid(normalizedCid(cidAsString));
+        Preconditions.checkArgument(!StringUtils.isBlank(cidAsString), "'cidAsString' is mandatory");
+        String normalizedCid = normalizedCid(cidAsString);
```

Comment:
I changed.
And because we still need checking null before using on unwrap method